client/my-sites/migrate/helpers/index.js

```diff
@@ -37,14 +37,13 @@ export function getImportSectionLocation( siteSlug, isJetpack = false ) {
 // Flow mapping dictionary, key is the path segment, value is the flow name
 const flowMapping = {
 	'import-focused': 'import-focused',
-	'import-hosted-site': 'import-hosted-site',
```

Comment:
This deletion here seems safe: looking at the `getImportFlowByURL` function below, other flows won't be impacted, and the function will still return `'onboarding-flow'`